stylesheets/base/_typography.scss

```diff
@@ -0,0 +1,118 @@
+/**
+ * Basic typography style for copy text
+ */
+
+body {
+  color: $text-color;
+  font: 400 14px / 28px $text-font;
+}
+.header-navigation {
+  color: $text-color;
+}
+.logotype h1{
+  color: $logo-color;
+  font: 700 30px / 34px $text-font;
+}
+.yellow-button{
+  color: $light-grey;
+  font: 600 16px / 24px $text-font;
+}
+.gray-button {
+  color: $logo-color;
+  font: 600 16px / 24px $text-font;
+}
+.main-headers {
+  color: $brand-color;
+  font: 700 44px / 54px $text-font;
+}
+.main-section-text p {
+  color: $paragraph-color;
+  font: 400 18px / 28px $text-font;
+}
+.playing-button {
+  font: 600 16px / 24px $text-font;
+  text-decoration: underline;
+}
+.name-section {
+  color: $brand-color;
+  font: 700 36px / 54px $text-font;
+}
+.design-tools-cards{
+  h2 {
+    color: $brand-color;
+    font: 600 22px / 33px $text-font;
+  }
+  p {
+    color: $paragraph-color;
+    font: 500 16px / 26px $text-font;
+  }
+}
+.section-in-two-text{
+  span {
+    color: $paragraph-color;
+    font: 500 16px / 24px $text-font;
+  }
+  h2 {
+    color: $brand-color;
+    font: 700 36px / 64px $text-font;
+  }
+  p, li {
+    color: $paragraph-color;
+    font: 500 16px / 24px $text-font;
+  }
+  h3 {
+    color: $brand-color;
+    font: 700 16px / 24px $text-font;
+  }
+
+}
+.section-hero{
+  h2 {
+    color: $brand-color;
+    font: 700 36px / 54px $text-font;
+  }
+  p {
+    color: $paragraph-color;
+    font: 500 16px / 26px $text-font;
+  }
+}
+.section-news {
```

Comment:
typography file is usually used for styling common text element (mostly titles, paragraphs, quotes)
all other section, buttons font styles should be where all other styles for this elements specified
here is an example of what is usually included in _typography.scss
https://gist.github.com/sebastiano-guerriero/2d05b319b9ae7618d99809797a67c0a0#file-typography_10-scss